server/routes/topics.js

```diff
@@ -13,14 +14,14 @@ const fixUrls = element => {
   }
 };
 
-const createTopicsRouter = ({ hubMenuService }) => {
+const createTopicsRouter = ({ topicsService }) => {
   const router = express.Router();
 
   router.get('/', async (req, res, next) => {
     try {
       const userName = req.user && req.user.getFullName();
       const establishmentId = path(['session', 'establishmentId'], req);
-      const topics = await hubMenuService.allTopics(establishmentId);
+      const topics = await topicsService.getTopics(establishmentId);
```

Comment:
minor, but you could have destructed out this `({ topicsService: { getTopics } })`

Reply:
kinda like having the reference to the service in the code where it's used. 

Can make it easier to find out where things are being used - we have some places where similar named methods are declared at different levels. I personally find it easier to see the context of where the function is declared in the code - we have quite a few functions with the same name declared in different functions.

It also helps evaluate cohesiveness easier if you can easily see which functions in a class use specific dependencies, rather than a detached function.